Jswope00/change external refs to internal (#759)
openedx/docs.openedx.org: source/site_ops/how-tos/enable_public_course_content.rst

.. _Enable Public Course Content:

####################################
Enabling Public Course Content
####################################

> By default, learners must create an Open edX account, be signed in to the LMS, 

Reviewer: ```suggestion
By default, learners must create an account on your Open edX instance, be signed in to the LMS, 
```

Not sure if this is the right phrasing but "Open edX account" might be misconstrued as an account on the Open edX sandbox or on openedx.org

and enroll in a course before they can see the course content. The *Public 
Course Content* feature gives you the option to make either a course outline 
or course content available to anyone, regardless of whether they have registered 

> for an Open edX account or enrolled in the course. You can decide which courses, 

Reviewer: ```suggestion
for an account on your instance or enrolled in the course. You can decide which courses, 
```

to your course outline or specific content blocks will be able to find your 
course.

#. View your course in Studio, and navigate to the **Advanced Settings** page. 

#. Locate the **Course Visibility in Catalog** setting and set it to ``"both"``.
   
   
********************
Feature Limitations
********************

The public course content feature is currently subject to some limitations, 
including the following.

* Anonymous or unenrolled learners are not members of any cohort, and so they
  only see course blocks that are not assigned to a content group. If you want
  to restrict public access to selected course blocks, you need create content
  groups for your private content, and ensure that your enrolled learners are
  members of a cohort that can see this content group.

* Only Text components, Video components, and course handouts have a "public"
  view. Unenrolled learners will see a message requesting that they enroll to 
  see more complex content types, such as discussion forums, problem blocks, 
  randomized content blocks, exams, Open Response Assessment, and other XBlocks.

* Unenrolled learners will not see course completion or progress updates as 

>   they proceed through the course, and Open edX doesn't remember where they 

Reviewer: ```suggestion
  they proceed through the course, and the LMS doesn't remember where they 
```

  left off if they leave the course and come back.

> * The edX mobile apps do not support public course content.

Reviewer: ```suggestion
* The Open edX mobile apps do not support public course content.
```
